tomlkit/container.py

```diff
@@ -46,6 +48,25 @@ def __init__(self, parsed: bool = False) -> None:
     def body(self) -> List[Tuple[Optional[Key], Item]]:
         return self._body
 
+    def unwrap(self) -> str:
+        unwrapped = {}
+        for k, v in self.items():
+            if k is None:
+                continue
+
+            if not type(k) == str:
```

Comment:
`type(k) == str` is not recommended against `isinstance`. For this check, we can replace to `isinstance(Key)`

Reply:
ok I have done that

now its that because of a possibility that `k` and `v` are tomlkit types  that inherit from `str`?

Reply:
I think I need to do some similar stuff with the tests that use the `type(..)` function. I will let you know when I get that done in the next day or so